packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -204,34 +252,24 @@ const SettingsRoutes = ({
   integration,
 }: { isSignedIn: boolean; integration: Integration } & RouteComponentProps) => {
   const location = useLocation();
-  // TODO: Remove this + config.sendFxAStatusOnSettings check once we confirm this works
-  const config = useConfig();
+  const isSync = integration != null ? integration.isSync() : false;
 
-  // Request and update account data/state to match the browser state because:
-  // 1) If a user logs in through the browser Sync pairing flow and then accesses
```

Comment:
Should we leave some form of this comment around?

Reply:
I'll try to incorporate in the main effect.